curator/api/utils.py

```diff
@@ -292,13 +292,27 @@ def prune_closed(client, indices):
     :arg indices: A list of indices to act on
     :rtype: list
     """
+    return prune_open_or_closed(client, indices, append_closed=False)
+
+def prune_opened(client, indices):
+    """
+    Return list of indices that are not open.
+
+    :arg client: The Elasticsearch client connection
+    :arg indices: A list of indices to act on
+    :rtype: list
+    """
+    return prune_open_or_closed(client, indices, append_closed=True)
+
+def prune_open_or_closed(client, indices, append_closed):
```

Comment:
You know what?  This is fine.  Disregard my previous comment.  We'll stick with this.